course/permissions.py

```diff
@@ -138,6 +136,11 @@ def has_object_permission(self, request, view, module):
             return module.are_requirements_passed(points)
         return True
 
+        if module.requirements.count() > 0:
```

Comment:
`if module.requirements.count() > 0:` is done two times in a row now. The second one should be removed, right?

Reply:
Done.